src/scripts/components/DropdownCheckbox.jsx

```diff
@@ -36,6 +36,16 @@ class DropdownCheckbox extends React.Component {
     }
   }
 
+  onCheckboxChange = () => {
+    const { checked } = this.state;
+    if (checked) {
+      this.setState({ activeKey: 1 });
+    } else {
+      this.setState({ activeKey: 0 });
+    }
+    this.props.onChange();
+  }
+
```

Comment:
We can pass active key as props from where we are calling this component. No meed to set activeKey here.